sklearn/preprocessing/data.py

```diff
@@ -244,6 +244,24 @@ class MinMaxScaler(BaseEstimator, TransformerMixin):
         .. versionadded:: 0.17
            *data_range_*
 
+    Examples
+    --------
+    >>> from sklearn.preprocessing import MinMaxScaler
+    >>>
+    >>> data = [[-1, -1], [-1, -1], [1, 1], [1, 1]]
```

Comment:
I think the example would be more informative with some intermediate values and different ranges:

```
>>> data = [[-1, -1], [-0.5, -0], [0, 3], [1, 5]]
```